consensus/types/src/beacon_block_body.rs

```diff
@@ -834,22 +839,16 @@ impl<E: EthSpec> BeaconBlockBody<E> {
             _ => return Err(Error::IndexNotSupported(generalized_index)),
         };
 
-        let attestations_root = match self {
-            BeaconBlockBody::Base(_)
-            | BeaconBlockBody::Altair(_)
-            | BeaconBlockBody::Bellatrix(_)
-            | BeaconBlockBody::Capella(_)
-            | BeaconBlockBody::Deneb(_) => self.attestations_base()?.tree_hash_root(),
-            BeaconBlockBody::Electra(_) => self.attestations_electra()?.tree_hash_root(),
+        let attestations_root = if self.fork_name() > ForkName::Electra {
```

Comment:
watch out here 😬 
```suggestion
        let attestations_root = if self.fork_name() >= ForkName::Electra {
```

Reply:
ah tricky.. maybe we should be adding functions to `ForkName` like `fork_name.electra_enabled()`

Reply:
Oh that would be better 👌

Reply:
opened a PR here https://github.com/dapplion/lighthouse/pull/36